README.md

```diff
@@ -76,17 +76,17 @@ To launch the HAT, follow these steps:
     ```
 2. Compile the project:
     ```bash
-    > sbt compile
+    > make dev
     ```
 3. Add custom local domain mapping to your `/etc/hosts` file. This will make sure when you go to the defined address from your machine you will be pointed back to your own machine. E.g.:
     ```
     127.0.0.1   bobtheplumber.hat.org

```

Comment:
They probably need to add the "bobtheplumber.example.com" in hosts, right?

Reply:
Fixed in the latest.  Thank you!